tests/claude_teams_surface_placement_test_support.py

```
@@ -0,0 +1,287 @@
+    return subprocess.run(
+        [cli_path, "--socket", str(socket_path), *args],
+        capture_output=True,
+        text=True,
+        check=False,
+        env=env,
+        timeout=30,
+    )
+
+
+def run_targetless_window_action_regressions(cli_path: str) -> None:
+    with tempfile.TemporaryDirectory(prefix="cmux-claude-teams-targetless-") as td:
```

Ruff (0.16.3)

[error] Undefined name `tempfile`

(F821)